jerry-core/vm/vm-defines.h

```diff
@@ -48,6 +48,7 @@ typedef struct
   ecma_value_t *stack_top_p;                          /**< stack top pointer */
   jmem_cpointer_t *literal_start_p;                   /**< literal list start pointer */
   ecma_object_t *lex_env_p;                           /**< current lexical environment */
+  struct vm_frame_ctx_t *prev_context_p;              /**< previous context */
```

Comment:
It is used now. Instead of a local variable it is stored as part of the structure.